TeamCode/src/main/java/org/firstinspires/ftc/teamcode/localization/Localizer.java

```diff
@@ -0,0 +1,25 @@
+package org.firstinspires.ftc.teamcode.localization;
+
+import com.acmerobotics.roadrunner.Pose2d;
+import com.acmerobotics.roadrunner.PoseVelocity2d;
+
+/**
+ * Interface for localization methods.
+ */
+public interface Localizer {
+    void setPose(Pose2d pose);
+
+    /**
+     * Returns the current pose estimate.
+     * NOTE: Does not update the pose estimate;
+     * you must call update() to update the pose estimate.
+     * @return the Localizer's current pose
+     */
+    Pose2d getPose();
+
+    /**
+     * Updates the Localizer's position estimate.
```

Comment:
Pose estimate is more accurate here than position estimate.

Reply:
done :)